pkg/nsx/services/subnet/subnet.go

```diff
@@ -155,26 +152,22 @@ func (service *SubnetService) createOrUpdateSubnet(obj client.Object, nsxSubnet
 			return nil, err
 		}
 	}
-	log.Info("Successfully updated nsxSubnet", "nsxSubnet", nsxSubnet)
+	log.Info("Successfully created or updated nsxSubnet", "nsxSubnet", nsxSubnet)
 	return nsxSubnet, nil
 }
 
 func (service *SubnetService) DeleteSubnet(nsxSubnet model.VpcSubnet) error {
-	vpcInfo, _ := common.ParseVPCResourcePath(*nsxSubnet.Path)
+	nsxOrgID, nsxProjectID, nsxVPCID, nsxSubnetID := nsxutil.ParseVPCPath(*nsxSubnet.Path)
```

Comment:
Would you think it's better to replace nsxutil.ParseVPCPath by common.ParseVPCResourcePath? It looks like ParseVPCResourcePath is a better implementation version of nsxutil.ParseVPCPath?

Reply:
+1

Reply:
good suggestion, Thanks. I changed to use common.ParseVPCResourcePath
and also remove sxutil.ParseVPCPath since it's not used any more.